### ISSUE TITLE
Support for detail

### ISSUE THREAD

**Issue by Hoffenbar** [h3rald/hastyscribe#87]
I've noticed that `<detail><summary>summary</summary>details</detail>`-blocks are principally supported, but if I display the generated html document in my browser, there is no icon/marker visible which should signal that there is an expandable block available when clicking on the summary:

![image](https://user-images.githubusercontent.com/46227007/145288373-27666561-a66d-40fb-92d4-abe874f71194.png)

If I render the same .md with mdbook, I notice almost the same html source code, but it looks like this (same browser!):

![image](https://user-images.githubusercontent.com/46227007/145288590-e933bb7d-3a68-42cc-bcc8-864d7b80df69.png)

It would be great, if such a marker would be visible in output generated by hastyScribe. Maybe I'm only missing something?
Additionaly, it would be even greater if all collapsed `<detail>`-blocks would be automatically expanded in case of printing (to pdf via browser). Due to my lack of knowledge about HTML, CSS and JavaScript and the like I'm not sure if it's possible at all.

**Comment by h3rald** [comment]
Yep, makes sense! I gotta do some tests and update the CSS... when I find some time. Patches welcome of course!

**Comment by h3rald** [comment]
Implemented in v2.0.0!
